lib/beaker/hypervisor/docker.rb

```diff
@@ -58,7 +58,7 @@ def host.enable_root_login(host,opts)
 
       # If the container is running ssh as its init process then this method
       # will cause issues.
-      if host[:docker_cmd] =~ /sshd/
+      if /sshd/.match?(host[:docker_cmd])
```

Comment:
I think this is really triggering failures on Ruby 3.2 because `docker_cmd` can be an array.

I haven't checked what's really the right thing to do here. Perhaps inspect the first argument if it's an array?